Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1201,4 +1246,479 @@ SPNs have a single credential and most scenarios that use them cannot support mu
 
 <br />
 
-___ 
\ No newline at end of file
+___ 
+
+## Azure_Subscription_Config_Enable_MicrosoftDefender_Servers
+
+### Display Name 
+Microsoft Defender for Servers should be enabled on subscriptions
+
+### Rationale 
+Microsoft Defender for servers provides real-time threat protection for your server workloads and generates hardening recommendations as well as alerts about suspicious activities.
+
+### Control Settings 
+```json 
+{
+    "ReqMDCTier": "Standard",
+    "ReqMDCTierResourceTypes": [
+        {
+            "Type": "VirtualMachines",
+            "DisplayName": "Servers"
+        }
+    ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> All required resource types are configured with MDC standard tier
+> 
+> **Failed:** 
+> Any of the below condition is satisfied:
+>   - Any of the required resource types is not configured with MDC standard tier.
+>   - MDC Tier is not enabled on the Subscription.
+>   - Fail if security center provider is not registered.
+>   - The access to the security center settings via REST API is denied (ErrorCode such as DisallowedOperations).
+
+
+### Recommendation 
+
+- **Azure Portal** 
+    
+    To enable this plan on all servers in your subscription: From Defender for Cloud's 'Environment settings' page, select the relevant subscription. In the 'Defender plans' page, set 'Servers' to 'On'."
+
+
+### Azure Policies or REST APIs used for evaluation 
+
+- API to get providers list with registration status for subscription level : - /subscriptions/{subscriptionId}/providers?api-version=2020-06-01&$select=namespace,registrationstate <br />
```

Comment:
API to get providers -> Rest API to get providers

Reply:
Fixed.